packages/roosterjs-content-model/lib/modelApi/image/applyImageBorderFormat.ts

```diff
@@ -0,0 +1,50 @@
+import { Border, extractBorderValues } from '../../domUtils/borderValues';
+import { ContentModelImage } from '../../publicTypes/segment/ContentModelImage';
+import { ptToPx } from '../../formatHandlers/utils/pointsToPixels';
+
+/**
+ * @internal
+ */
+export default function applyImageBorderFormat(
+    image: ContentModelImage,
+    border: Border,
+    isPt?: boolean
+) {
+    const format = image.format;
+    const { width, style, color } = border;
+    const borderKey = 'borderTop';
+    const extractedBorder = extractBorderValues(format[borderKey]);
+    const borderColor = extractedBorder.color;
+    const borderWidth = extractedBorder.width;
+    const borderStyle = extractedBorder.style;
+    let borderFormat = '';
+
+    if (width) {
```

Comment:
Is 0 also a valid value?

Reply:
No, the width is a string, it is passed the value + unit. Ex: 15px

Reply:
ok